administrator/components/com_scheduler/tmpl/tasks/default.php

```diff
@@ -0,0 +1,206 @@
+<?php
+/**
+ * @package         Joomla.Administrator
+ * @subpackage      com_scheduler
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ *
+ * phpcs:ignoreFile
+ */
+
+// Restrict direct access
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Layout\LayoutHelper;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Session\Session;
+use Joomla\Component\Scheduler\Administrator\View\Tasks\HtmlView;
+
+/** @var  HtmlView  $this*/
+
+HTMLHelper::_('behavior.multiselect');
+
+try
+{
+	$app = Factory::getApplication();
+} catch (Exception $e)
+{
+	die('Failed to get app');
+}
+
+$user = $app->getIdentity();
+$userId = $user->get('id');
+$listOrder = $this->escape($this->state->get('list.ordering'));
+$listDirn = $this->escape($this->state->get('list.direction'));
+$saveOrder = $listOrder == 'a.ordering';
+$section = null;
+$mode = false;
+
+
+if ($saveOrder && !empty($this->items))
+{
+	$saveOrderingUrl = 'index.php?option=com_scheduler&task=tasks.saveOrderAjax&tmpl=component&' . Session::getFormToken() . '=1';
+	HTMLHelper::_('draggablelist.draggable');
+}
+?>
+
+<form action="<?php echo Route::_('index.php?option=com_scheduler&view=tasks'); ?>" method="post" name="adminForm"
+	  id="adminForm">
+	<div id="j-main-container" class="j-main-container">
+		<?php
+		// Search tools bar
+		echo LayoutHelper::render('joomla.searchtools.default', array('view' => $this));
+		?>
+
+		<!-- If no tasks -->
+		<?php if (empty($this->items)): ?>
+			<!-- No tasks -->
+			<div class="alert alert-info">
+				<span class="icon-info-circle" aria-hidden="true"></span><span
+						class="visually-hidden"><?php echo Text::_('INFO'); ?></span>
+				<?php echo Text::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
+			</div>
+		<?php endif; ?>
```

Comment:
This can be automatically done by the searchtools + string in the filter XML for the search field.